src/commands/compare.ts

```diff
@@ -38,8 +38,8 @@ export default class Compare extends Command {
     const { currentBranch, targetBranch } = localFlags;
     const checker = new BundleChecker(localFlags);
     const report = await checker.compare();
-    const overviewReportHeader = ['File name', targetBranch, currentBranch] as ITableRow;
-    const filesBreakDownHeader = ['File extension', targetBranch, currentBranch] as ITableRow;
+    const overviewReportHeader = ['File name', currentBranch, targetBranch] as ITableRow;
+    const filesBreakDownHeader = ['File extension', currentBranch, targetBranch] as ITableRow;
```

Comment:
targetBranch is `master` - are we moving it to the third column?

Reply:
I prefer to have the `current branch` in 2nd columns since the delta is associated with his changes